climada/util/coordinates.py

```diff
@@ -76,6 +78,256 @@
 MAX_DEM_TILES_DOWN = 300
 """Maximum DEM tiles to dowload"""
 
+######### copied over from interpolation module #####################
```

Comment:
Does this mean that nothing in this pull request is new?

Reply:
This means that i copied this part from the interpolation method (until where the next full line of ######### is). It's just a marker to facilitate the review. I will of course delete this remark later.